enos/README.md

```diff
@@ -13,17 +13,12 @@ for further information regarding installation, execution or composing Enos scen
 ## Requirements
 * AWS access. HashiCorp Boundary developers should use Doormat.
 * Terraform >= 1.0
-* Enos >= v0.0.10. You can [install it from a release channel](https://github.com/hashicorp/Enos-Docs/blob/main/installation.md) or use `make tools` install it into `$GOBIN`.
-* Access to the QTI org in Terraform Cloud. HashiCorp Boundary developers can
-  access this token in 1Password or request their own in #team-quality on slack.
+* Enos >= v0.0.10 (`brew tap hashicorp/tap && brew update && brew install hashicorp/tap/enos`)
```

Comment:
Oh boy, 0.0.10 is ancient. I think the first public version was 0.0.28.

Reply:
I'll update that.

Also, I need to update our reference to not use the qti vault module.